spec/rubocop/cop/style/space_inside_block_braces_spec.rb

```diff
@@ -22,6 +22,17 @@
       expect(cop.messages).to be_empty
     end
 
+    it 'accepts braces with something inside' do
+      inspect_source(cop, 'each { "f" }')
+      expect(cop.messages).to be_empty
```

Comment:
It turns out most of these tests are checking if `cop.messages` or `cop.highlights` is empty. I have updated all of the tests.